metadata-comparison/src/main/java/org/codice/nitf/metadatacomparison/FileComparer.java

```diff
@@ -813,21 +780,24 @@ private void compareMetadataFiles() {
             }
             LOGGER.debug("  * Done");
         } else {
-            new File(filename + THEIR_OUTPUT_EXTENSION).delete();
-            new File(filename + OUR_OUTPUT_EXTENSION).delete();
+            try {
+                Files.delete(Paths.get(filename + THEIR_OUTPUT_EXTENSION));
+                Files.delete(Paths.get(filename + OUR_OUTPUT_EXTENSION));
+            } catch (IOException e) {
+                LOGGER.error("Error deleting {}: {}", filename, e);
+            }
         }
     }
 
     private static List<String> fileToLines(String filename) {
         List<String> lines = new LinkedList<>();
         String line;
-        try {
-                BufferedReader in = new BufferedReader(new java.io.FileReader(filename));
+        try (BufferedReader in = new BufferedReader(new java.io.FileReader(filename))) {
```

Comment:
This method looks like it should have been closing out this reader after gathering the lines. Will revert if I misinterpreted and the stream should be kept open.